packages/react-core/src/components/LoginPage/LoginMainHeader.tsx

```diff
@@ -12,13 +12,16 @@ export interface LoginMainHeaderProps extends React.HTMLProps<HTMLDivElement> {
   title?: string;
   /** Subtitle that contains the Text, URL, and URL Text for the Login Main Header */
```

Comment:
out of scope but what does URL and URL Text refer to? This is a generic "description" element in core, and rendered as a `<p>`. Seems like it might be worth updating this prop to `description` (like the [modal component](https://www.patternfly.org/v4/components/modal#modal)) since `subtitle` implies it might be a heading element or something - it would be invalid invalid to pass a heading there currently since you can't put a heading in a `<p>`. On that, we might also consider just making this a `<div>` in both core and react. If that seems valid, I can open an issue.

Reply:
I see what you mean. I could definitely see the user placing a hyperlink URL here. If you do open up that issue, feel free to tag me so I can make a react issue to update the prop name and usage